docs/quantize/quantize_example.py

```diff
@@ -0,0 +1,104 @@
+from functools import partial
+
+import tensorflow as tf
+
+from blueoil.networks.<TASK_NAME>.<NETWORK_NAME> import FooNetwork
```

Comment:
We want a small example network. Not template.
It's ok to assume task as classification in this case.

Reply:
Please review this in #966 instead.

Reply:
This change to classification task and put in `blueoil/networks/classification` in #966 .

Reply:
Thanks!! I'll review the sample code PR.